src/containers/offer-page/teaching-block/TeachingBlock.tsx

```diff
@@ -51,6 +53,8 @@ const TeachingBlock = <T extends CreateOrUpdateOfferData>({
     }
   }
 
+  const studyMaterials = userRole === UserRoleEnum.Student && <StudyMaterials />
```

Comment:
May be it is better not to create a variable but just move `userRole === UserRoleEnum.Student && <StudyMaterials />` to 140 line and put it in brackets instead of `studyMaterials`. It is not necessary, it's up to you